Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.m

```diff
@@ -121,8 +122,8 @@ - (void)adjustBarConstraintsForView:(UIView *)view {
         [NSLayoutConstraint deactivateConstraints:self.barConstraints];
     }
     self.barConstraints = @[
-        [self.barView.leadingAnchor constraintEqualToAnchor:view.leadingAnchor constant:-BPKSpacingBase],
```

Comment:
Gonna handle this in the item component itself as that'll give a better tap area